docs/BreakingChanges/040 ObjectContext.CreateDatabase and DbProviderService.md

```diff
@@ -0,0 +1,23 @@
+## 40: ObjectContext.CreateDatabase and DbProviderServices.CreateDatabase methods
```

Comment:
{
  "Id": "40",
  "Title": "ObjectContext.CreateDatabase and DbProviderServices.CreateDatabase methods",
  "Details": "Beginning in .NET 4.5, if database creation fails, `CreateDatabase` methods will attempt to drop the empty database. If that operation succeeds, the original `SQLException` will be propagated (instead of the `InvalidOperationException` that was always thrown in .NET 4.0)",
  "Suggestion": "When catching an InvalidOperationException while executing ObjectContext.CreateDatabase or DbProviderServices.CreateDatabase, SQLExceptions should now also be caught.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#sql",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.Objects.ObjectContext.CreateDatabase",
    "M:System.Data.Common.DbProviderServices.CreateDatabase(System.Data.Common.DbConnection,System.Nullable{System.Int32},System.Data.Metadata.Edm.StoreItemCollection)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Data.Objects.ObjectContext.CreateDatabase
- [x] M:System.Data.Common.DbProviderServices.CreateDatabase(System.Data.Common.DbConnection,System.Nullable{System.Int32},System.Data.Metadata.Edm.StoreItemCollection)